package.json

```diff
@@ -48,7 +49,9 @@
         "react-dom": "^17.0.1",
         "react-helmet": "^6.1.0",
         "sass": "^1.49.9",
+        "sharp": "^0.29.3",
```

Comment:
just wondering if there was a reason sharp, babel-eslint, and typescript are being added?

Reply:
For this one: these were things that my computer was missing so it got added to the package.json in order for me to run it. I can remove it if you want!